connector/datadogconnector/metadata.yaml

```
@@ -0,0 +1,9 @@
+type: datadog
+
+status:
+  class: connector
+  stability:
+    alpha: [traces_to_metrics]
+  distributions: [contrib]
```

For now, this is not on the contrib distribution, let's leave this for after we make a PR on opentelemetry-collector-releases and actually add this to the contrib distro.
```suggestion
  distributions: []
```

---

This is not resolved.